.gitattributes

```diff
@@ -0,0 +1,5 @@
+* text=auto
+
+*.ts text eol=lf
+*.sh text eol=lf
+*.json text eol=lf
```

Comment:
Let's keep Linux file endings.

Reply:
I assume it makes sense to have a .gitattributes file to make the repo work for whoever checks it out on whatever system rather than expecting global git settings not to be set to auto?
Probably make the default to be  `text eol=lf` and then call out binaries?

Reply:
I'm fine with that; but perhaps let's take that to a separate PR?